test/module-test.cc

```diff
@@ -405,9 +411,9 @@ TEST(module_test, color) {
 
 TEST(module_test, cstring_view) {
   fmt::cstring_view nsv("fmt");
-  EXPECT_EQ("fmt", nsv.c_str());
+  EXPECT_EQ("fmt", std::string_view(nsv.c_str()));
```

Comment:
I suggest using `EXPECT_STREQ` instead.

Reply:
Actually no, this checks the pointer comparison so the correct fix is to move "fmt" into a separate variable.

Reply:
My new code cannot fall into the identity trap anymore.

Reply:
Not sure what you mean by identity trap but this must compare pointers, not strings, something like:
```c++
auto s = "fmt";
EXPECT_EQ(s, fmt::cstring_view(s).c_str());
```